controller/generate_model.py

```diff
@@ -2,13 +2,17 @@
 # SPDX-License-Identifier: MIT-0
 
 from jinja2 import Environment, FileSystemLoader
-
+import os
 class CreateScript:
     """
         Generates model classes
     """
     def __init__(self, filename, code):
-        self.script_name = 'models/'+filename + '.py'
+        folder = 'models'
```

Comment:
can you add this is a config file?

Reply:
Sure, it sounds great. I add a config.ini file and configure both the folder path and file name